framework/material/Form.kt

```diff
@@ -514,7 +525,7 @@ object Form {
                         .padding(vertical = MULTILINE_INPUT_PADDING)
                         .horizontalScroll(state.horScroller)
                 ) {
-                    Row(Modifier.align(alignment = Alignment.CenterStart)) {
+                    Row(Modifier.align(alignment = Alignment.TopStart)) {
```

Comment:
This `Row` no longer does anything, and can be deleted.

Reply:
Done.